src/openms/include/OpenMS/QC/Ms2IdentificationRate.h

```diff
@@ -66,45 +66,43 @@ namespace OpenMS
     /// container that stores results
     std::vector<IdentificationRateData> rate_result_;
 
-    /// struct that is made to store results
-    IdentificationRateData id_rate_data_;
-
     /**
      * @brief counts peptideidentifications
      * used to count assigned and unassigned peptideidentifications in a featuremap
-     * @param peptide_id
-     * @param force_fdr
+     * @param peptide_id - vector of peptideidentifications
+     * @param force_fdr - bool for forceflag, if it's true all peptides are count despite fdr was not made
      * @return number of peptideidentifications in a given vector of peptideidentifications
      * @exception Exception::Precondition is thrown if there wasn't made a FDR before
      * @warning LOG_WARN if there is a peptideidentification without peptidehits
      */
-    Int64 countPeptideId_(std::vector<PeptideIdentification> peptide_id, bool force_fdr);
+    Int64 countPeptideId_(const std::vector<PeptideIdentification>& peptide_id, bool force_fdr);
 
   public:
     /// Default constructor
-    Ms2IdentificationRate();
+    Ms2IdentificationRate() = default;
 
     /// Destructor
-    ~Ms2IdentificationRate();
+    ~Ms2IdentificationRate() = default;
 
     /**
      * @brief computes Ms2 Identification Rate
      * stores results as a struct in a vector
-     * @param feature_map
-     * @param exp
-     * @param force_fdr
+     * @param feature_map - FeatureMap
```

Comment:
No need for `-`. Just write `@param feature_map Input featuremap with target/decoy annotation`